packages/graphviz/fix-edgepaintmain-include.patch

```diff
@@ -0,0 +1,11 @@
+--- graphviz-2.38.0/cmd/edgepaint/edgepaintmain.c	2014-04-13 13:40:25.000000000 -0700
++++ src/cmd/edgepaint/edgepaintmain.c	2016-04-03 10:46:38.532882314 -0700
+@@ -27,7 +27,7 @@
+ #include <cgraph.h>
+ #include <agxbuf.h>
+ #include <ingraphs.h>
+-#include <pointset.h>
++#include "pointset.h"
```

Comment:
All changes were necessary at the time I wrote these patches. After madly hacking away to get Graphviz to compile, I removed any changes that wouldn't break the build.